docs/evm-compatibility.md

```diff
@@ -290,7 +290,27 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_call","params":[{see above}]
 - **Returns**: `String` - The estimated gas amount.
 
 ```bash
-curl -X POST --data '{"jsonrpc":"2.0","method":"eth_estimateGas","params":[{see above}],"id":1}'
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_estimateGas","params":[{"from":"0xFF3014B077D307E7B0bf262d072B25dbE19E2Be3","to":"0xd3CdA913deB6f67967B99D67aCDFa1712C293601","value":"0x186a0"}],"id":1}' -H "Content-Type: application/json" https://rpc.test.mezo.org
+```
+
+#### eth_estimateCost
+
+- **Description**: Estimates the cost necessary to execute a transaction.
+- **Parameters**:
+    Object:
+    - `from`: DATA, 20 Bytes - The address the transaction is send from.
+    - `to`: DATA, 20 Bytes - (optional when creating new contract) The address the transaction is directed to.
+    - `value`: QUANTITY - value sent with this transaction
+- **Returns**:
+    Object:
+    - `precision` - The precision cost values are presented with.
```

Comment:
This is typically referred to as `decimals`, e.g. in ERC20 setups. Thoughts on sticking to that terminology?

Reply:
Yeah good point, changed in 547c54dac923dce1f6a94675c5e5160e78b502a2